spyder/app/tour.py

```diff
@@ -584,9 +611,13 @@ def toolbutton(icon):
 
         arrow = get_image_path('hide.png')
 
+        self.color_top = color_top
+        self.color_back = color_back
+        self.combobox_background = combobox_background
         self.stylesheet = '''QComboBox {
                              padding-left: 5px;
-                             background-color: rgbs(230,230,230,100%);
+                             background-color:
+                          ''' + self.combobox_background.name() + '''

```

Comment:
This is ugly. Can't we use string replacement here? I mean, something like

    ...
    background-color: {}
    ...
    '''.format(self.combobox_background.name())

And let's try to do the same with the `arrow` variable used below in this same string.